kimchi/src/prover.rs

```diff
@@ -821,39 +821,26 @@ where
         //~ 1. If lookup is used, evaluate the following polynomials at $\zeta$ and $\zeta \omega$:
         if index.cs.lookup_constraint_system.is_some() {
             //~~ - the aggregation polynomial
-            let aggreg = lookup_context
-                .aggreg_coeffs
-                .as_ref()
-                .unwrap()
-                .to_chunked_polynomial(index.max_poly_size);
+            let aggreg = lookup_context.aggreg_coeffs.as_ref().unwrap();
```

Comment:
What is now the flow? Don't we check anymore that the polynomials fit in the max_poly_size?

Reply:
I don't think we ever performed that check, but one thing we still need to do is enforce the length of the commitments we receive